api4/schema.graphqls

```diff
@@ -85,24 +90,40 @@ type ChannelMember {
 type User {
 	id: String!
 	username: String!
+	# deliberately omitting password
 	email: String!
+	authData: String
 	firstName: String!
 	lastName: String!
 	nickname: String!
+	emailVerified: Boolean!
 	isBot: Boolean!
 	isGuest: Boolean!
 	isSystemAdmin: Boolean!
 	createAt: Float!
+	updateAt: Float!
 	deleteAt: Float!
 	authService: String!
 	customStatus: CustomStatus
 	status: Status
 	props: StringMap!
 	notifyProps: StringMap!
-	lastPictureUpdateAt: Float!
+	lastPictureUpdate: Float!
+	lastPasswordUpdate: Float!
+	failedAttempts: Float!
```

Comment:
Also, this one seems to be passed down, but I'm surprised it's also not kept to the server only

Reply:
This does not seem to be sanitized, so keeping it.